### DISCUSSION TITLE
Ialarm-xr antifurto365 integrazioni o altro

### DISCUSSION THREAD

**Discussion by dodamerenda** [indomus/forum#1428] [HUB: Home Assistant]
Salve 
sono nuova qui, intanto ...
mi piacerebbe saper se esiste un qualche modo per integrare la centralina **Ialarm-xr** .. o anche solo il sistema di sensori e rivelatori configurati.
avevo pensato ad utilizzare l'integrazione ufficiale .. ma è indicato **non** funzioni con **XR** .. 
cosa suggerite? 
**aiuto**
grazie !

**Comment by snakuzzo** [comment]
Ciao.
Hai provato a cercare?

https://github.com/indomus/forum/discussions?discussions_q=ialarm

**Reply by dodamerenda** [reply]
grazie 
guardai .. ma a parte un idea di servirmi di MQTT per armo e disarmo .. e qualche considerazione sull'integrazione per la versione non XR ... non ho trovato altro  .. uffi

**Reply by snakuzzo** [reply]
Che io sappia la situazione purtroppo è questa. Prova comunque a chiedere negli altri topic già aperti

**Comment by bigmoby** [comment]
Ciao ragazzi,
partendo dall'ottimo lavoro di @RyuzakiKK ho fatto un pò di reverse engineering del nuovo protocollo usato da iAlarmXR ed ho creato la libreria https://github.com/bigmoby/pyialarmxr
Le invocazioni al servizio funzionano ma sto ancora lavorando per richiedere l'inclusione della libreria in Home Assistant.
Spero vi possa essere d'aiuto 👍

**Reply by gimalugi** [reply]
Ciao scusa la domanda magari stupida, ma sono nuovo qui. Sai quando sarà disponibile nelle repository di HA? C'è forse un altro modo di installarla? Grazie

**Reply by bigmoby** [reply]
Ciao @gimalugi , nessun problema. Al momento c'è una PR aperta in approvazione (https://github.com/home-assistant/core/pull/67817), il codice è tutto ok ma mancano, appunto, le approvazioni per andare sul branch main e staccare la release. Il quando non te lo so dire poiché è tutto nelle mani dei maintainer del repository di Home Assistant. Se vuoi provare l'implementazione  in ambiente di sviluppo locale potresti usare il fork dal mio repository (https://github.com/bigmoby/home-assistant/tree/feature/ialarmxr).

**Comment by gimalugi** [comment]
Ciao, grazie mille per il tuo lavoro.....

Il mar 22 mar 2022, 08:44 BigMoby ***@***.***> ha scritto:

**Comment by percevalknight** [comment]
Ciao, novità? Possiamo fare qualcosa per favorire l’ufficializzazione?
Come faccio inoltre a testare la libreria intanto? C’è un breve manuale?

**Reply by bigmoby** [reply]
Ciao @percevalknight la PR è ancora in review, purtroppo il processo è lento per mancanza di reviewers: https://github.com/home-assistant/core/projects/4#card-78726842

**Comment by bigmoby** [comment]
Ciao ragazzi, poiché il processo di revisione ufficiale sta purtroppo andando un pò lungo per mancanza di reviewers, ho deciso di creare questo progetto di custom component della stessa integrazione a iAlarmXR: https://github.com/bigmoby/ialarmxr-custom-component
Attenzione: il progetto di integrazione custom rimarrà non più mantenuto una volta completato il processo di approvazione in Home Assistant core! 😉
Spero che questo progetto sia apprezzato 👍

**Comment by gimalugi** [comment]
Ciao, volevo avvisare di un problema riguardo  l'integrazione custom
ialarmxr.... è andato tutto moltobene fino allo scorso week end quando lo
staff di antifurto 365 ha eseguito un operazione di manutenzione ai
server...
da quel momento ho continui disconnessioni dal server. Ho disattivato
l'integrazione e tutto è tornato ok....solo per informarvi.

Il gio 21 apr 2022, 19:51 BigMoby ***@***.***> ha scritto:

**Comment by bigmoby** [comment]
Ciao @gimalugi , ti ringrazio per la segnalazione. Quindi al momento non ti è possibile utilizzare l'integrazione altrimenti ti si disconnette la mobile app? Potresti spiegarmi per cortesia meglio la problematica in modo che posso comprendere come poter risolvere?
Grazie mille 👍

**Comment by gimalugi** [comment]
Ciao, dunque succede questo:
Ad integrazione attivata, in tempi sempre diversi ma frequenti( non più di
10minuti) l'applicazione ialarm-xr si disconnette  e quindi anche
l'integrazione stessa e non si può più interagire con la centralina, anche
se la stessa continua a funzionare localmente, o meglio la centralina si
può comunque allarmare da tastierino, numerico da telecomando etc.
Ho disconnesso oggi mezzogiorno l'integrazione e l'app ialarm-xr si  è
subito riconnessa e non si è più disconnessa. Non so se sono stato chiaro
altrimenti chiedi pure cercherò di aiutarti a capire. Comunque grazie mille

Il mer 4 mag 2022, 15:23 BigMoby ***@***.***> ha scritto:

**Comment by gimalugi** [comment]
Scusami mi sono dimenticato di dirti che ho provato a disconnettere e a
riconnettere l'integrazione più volte ed è sempre successa la stessa cosa.

Il mer 4 mag 2022, 16:01 Gino Villa ***@***.***> ha scritto:

**Comment by bigmoby** [comment]
https://github.com/bigmoby/ialarmxr-custom-component/issues/1#issuecomment-1117829781

**Comment by bigmoby** [comment]
Buongiorno,
in questi giorni ho contattato Antifurto365 per risolvere la problematica riscontrata sul controllo della centrale iAlarmXR. Ringrazio lo Staff di Antifurto365 per il supporto, anche se al momento la mia situazione non è migliorata. Vi riporto qui di seguito quanto riferitomi da Antifurto365:

```
[...]
Si è trattato di un aggiornamento, non di un problema. Finalizzato a migliorare le prestazioni. 

[...] l'aggiornamento è terminato e tutto è tornato regolare. Ti parlo a nome mio e di altre migliaia di clienti che utilizzano (ed utilizziamo ) l'app ogni giorno.

Si è trattato di un aggiornamento eseguito tra il 30 aprile ed il 02 Maggio, adesso concluso.
[...]
```

Per quanto riguarda l'integrazione con Home Assistant, la cui implementazione sto portando avanti, vedrò nei prossimi giorni di diminuire il tempo di polling sul server. Ad oggi osservo molti errori di socket timeout sulla connessione sia in fase di instaurazione della connessione sia in fase di lettura della response. La mobile app (v1.4.8) ho analizzato che effettua il polling ogni 40 secondi, a tal proposito vi chiederei conferma anche se per voi notate lo stesso comportamento! Pertanto vedrò di effettuare, lato integrazione, un polling in linea con quanto fatto dalla mobile app. Lo dico con rammarico poiché prima dell'aggiornamento di Antifurto365, l'aggiornamento dello stato della centrale sulla mobile app era immediato e non con questo delay, vi chiedo anche per questo un vostro feedback ovvero se riscontrate lo stesso comportamento.
Per quanto riguarda gli sviluppi vedrò in futuro se mi sarà possibile creare una integrazione che comunichi con la centrale in locale ovvero escludendo la parte p2p. Non avendo nessuna specifica sulle API procederò, come finora ho fatto, operando di reverse engineering sul protocollo di rete con tutti i limiti e difficoltà del caso.
Saluti 👍

**Comment by gimalugi** [comment]
Hi Bigmoby my mobile app version is 1.5.3. if you Need a beta tester I'm
here.....😉

Il ven 6 mag 2022, 11:16 BigMoby ***@***.***> ha scritto:

**Reply by bigmoby** [reply]
> Hi Bigmoby my mobile app version is 1.5.3. if you Need a beta tester I'm here.....😉 Il ven 6 mag 2022, 11:16 BigMoby ***@***.***> ha scritto:
> […](#)
> Buongiorno, in questi giorni ho contattato Antifurto365 per risolvere la problematica riscontrata sul controllo della centrale iAlarmXR. Ringrazio lo Staff di Antifurto365 per il supporto, anche se al momento la mia situazione non è migliorata. Vi riporto qui di seguito quanto riferitomi da Antifurto365: [...] Si è trattato di un aggiornamento, non di un problema. Finalizzato a migliorare le prestazioni. [...] l'aggiornamento è terminato e tutto è tornato regolare. Ti parlo a nome mio e di altre migliaia di clienti che utilizzano (ed utilizziamo ) l'app ogni giorno. Si è trattato di un aggiornamento eseguito tra il 30 aprile ed il 02 Maggio, adesso concluso. [...] Per quanto riguarda l'integrazione con Home Assistant, la cui implementazione sto portando avanti, vedrò nei prossimi giorni di diminuire il tempo di polling sul server. La mobile app (v1.4.8) ho analizzato che effettua il polling ogni 40 secondi, a tal proposito vi chiederei conferma anche se per voi notate lo stesso comportamento! Pertanto vedrò di effettuare, lato integrazione, un polling in linea con quanto fatto dalla mobile app. Lo dico con rammarico poiché prima dell'aggiornamento di Antifurto365, l'aggiornamento dello stato della centrale sulla mobile app era immediato e non con questo delay, vi chiedo anche per questo un vostro feedback ovvero se riscontrate lo stesso comportamento. Per quanto riguarda gli sviluppi vedrò in futuro se mi sarà possibile creare una integrazione che comunichi con la centrale in locale ovvero escludendo la parte p2p. Non avendo nessuna specifica sulle API procederò, come finora ho fatto, operando di reverse engineering sul protocollo di rete con tutti i limiti e difficoltà del caso. Saluti 👍 — Reply to this email directly, view it on GitHub <[#1428 (comment)](https://github.com/indomus/forum/discussions/1428#discussioncomment-2699335)>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AYFKO7ROTXE4RXUTG7EOABDVITPN7ANCNFSM5NG2VBQQ> . You are receiving this because you were mentioned.Message ID: ***@***.***>

I updated my app to v1.5.3 but the same polling time as the previous: 40 seconds to update the status.

**Comment by bigmoby** [comment]
Buongiorno ragazzi,
vi chiedo la cortesia di non usare ne l'integrazione core per iAlarm XR ne tramite il custom component poiché Antifurto356 mi ha contattato e mi ha riferito che sta avendo dei problemi seri sul loro server. Non voglio entrare nel merito della questione (il mio codice è libero ed accessibile e ognuno se ne ha voglia può visionarne la sua struttura e farsi la propria opinione) ma dato che non voglio passare guai legali, perché purtroppo di questo stiamo parlando, vi esorto a rimuovere tale integrazione e ad utilizzare solo ed esclusivamente gli strumenti approvati e diffusi da Antifurto365.
Ieri sera ho creato una PR di rimozione da Home Assistant per chiudere il caso con Antifurto365 (https://github.com/home-assistant/core/pull/73083) ed inoltre ho dovuto chiudere sia il progetto del custom component sia la libreria di protocollo.
Certo della vostra comprensione,
Fabio

p.s.: la mia buona fede l'ho dimostrata da principio allineando sempre tutti.

**Reply by percevalknight** [reply]
Ciao, è un peccato, comunque ho rimosso le integrazioni.

**Reply by bigmoby** [reply]
> Ciao, è un peccato, comunque ho rimosso le integrazioni.

Si, un vero peccato...ti ringrazio molto @percevalknight

**Comment by bigmoby** [comment]
Integrazione rimossa: https://github.com/home-assistant/core/pull/73083
Due installazioni totali...no comment

**Comment by gabriele-latino** [comment]
Questo e' un chiarissimo esempio di come un'azienda possa essere chiusa mentalmente, e darsi la zappa sui piedi pensando di fare la voce grossa invece di favorire un miglioramento del proprio prodotto, grazie a sviluppatori che lavorano gratuitamente. Se avessi dovuto scegliere un sistema di allarme, oggi come oggi questo sarebbe stato sicuramente depennato dalla lista, non c'e' dubbio.

**Reply by bigmoby** [reply]
@gabriele-latino concordo su tutta la linea. Purtroppo ho anche rischiato di fare personalmente una figura pessima nei confronti della community dovendo ritirare l'integrazione pochissimi giorni dopo il rilascio. Nei giorni in cui ho ricevuto minacce di azioni legali ho scritto privatamente a Paulus Schoutsen spiegando l'accaduto nei dettagli ed ha capito subito la situazione in cui mi sono trovato dandomi immediatamente supporto. Non lo ringrazierò mai abbastanza! 
Ho dovuto rimuovere purtroppo anche il progetto di custom components... 🤐
Forza e coraggio, alla prossima, ragazzi! 🙌

**Reply by maxill1** [reply]
> @gabriele-latino concordo su tutta la linea. Purtroppo ho anche rischiato di fare personalmente una figura pessima nei confronti della community dovendo ritirare l'integrazione pochissimi giorni dopo il rilascio. Nei giorni in cui ho ricevuto minacce di azioni legali ho scritto privatamente a Paulus Schoutsen spiegando l'accaduto nei dettagli ed ha capito subito la situazione in cui mi sono trovato dandomi immediatamente supporto. Non lo ringrazierò mai abbastanza! Ho dovuto rimuovere purtroppo anche il progetto di custom components... 🤐 Forza e coraggio, alla prossima, ragazzi! 🙌

Scusa la domanda, non ho guardato il codice, ma pyialarmxr si connette direttamente alla centrale via TCP/IP usando il protocollo di wildstray? o passi per i loro server (AWS per altro)?

Perché nel primo caso capisco se fanno storie per il marchio, nel secondo potrebbero effettivamente dire che il traffico gli causa dei costi se usi il polling su server.

La cosa ridicola è che questo fantastico prodotto pubblicizzato come "europeo" e certificato in Germania è un rebranding con qualche personalizzazione/geolocalizzazione dei Meian/Focus.

Trovo anche io di cattivissimo gusto la richiesta di rimozione con minaccia di azioni legali.

Col senno del poi avrei preso un Meian/Focus pagandolo probabilmente la metà o avrei optato per un antifurto di altro marchio, meglio ancora se ZigBee.

News su un eventuale rimozione invece di iAlarm "normale"? Li non hanno detto nulla?

**Reply by bigmoby** [reply]
ciao @maxill1 si va in cloud (`"iot_class": "cloud_polling",`) ...dai pure un occhio al codice (ho dovuto chiudere pure il questo progetto)...protocollo di wildstray 👍 ...in cloud...e le sorprese ti potrebbero non mancare!
Per il marchio non avrebbero potuto fare storie poiché in tutti i progetti ho sempre specificato a chiedere lettere che non ero associato al marchio e che erano tutti registrati...ad ogni modo, amen.

Sulla libreria iAlarm "normale" non ho notizie, magari sarebbe curioso domandare a @RyuzakiKK

**Reply by RyuzakiKK** [reply]
In questi anni io non sono mai stato contattato da nessuno. Considerando che `pyialarm` funziona in locale, non avviene alcuno scambio di messaggi con il loro cloud. Quindi non c'è molto di cui si possano lamentare, è una semplice reimplementazione a scopo di interoperabilità con altri sistemi (HA), perfettamente lecito secondo l'ordinamento Italiano.

Nel caso facessero storie per il nome, anche se credo sia semplice fair use, posso sempre rinominarlo in pymeian.

**Reply by bigmoby** [reply]
Ciao @RyuzakiKK grazie molte per il tuo contributo! Credo anche io effettivamente che il problema risieda tutto nella comunicazione tramite il cloud e nel protocollo di comunicazione usato.
Purtroppo adesso non potrò neanche solo tentare di sviluppare una versione della libreria che funzioni in locale avendomi proposto caldamente di sostituire la mia centralina da XR a quella "normale"...ed io ho accettato pur di non aver altre rogne con loro. Quindi al momento sto usando la tua libreria @RyuzakiKK ...mi sto permettendo di smanettarci sopra (su un mio progetto privato...per carità!) per avere la gestione dei sensori andati in disconnessione o in allarme. Grazie ancora @RyuzakiKK

**Reply by RyuzakiKK** [reply]
> avendomi proposto caldamente di sostituire la mia centralina da XR a quella "normale"

Potresti spiegare meglio questo punto? Ti hanno proposto loro di passare alla versione precedente per permetterti di usare l'integrazione su HA?

**Reply by bigmoby** [reply]
Premetto che non posso scrivere apertamente in un canale pubblico, sono tuttora con una pratica ancora aperta dalla loro amministrazione per la valutazione di una azione legale.
Mi hanno cortesemente rimborsato della differenza di prezzo e spedito la precedente centrale d'allarme, in cambio della centrale XR.
A me sta bene così se non ci saranno ulteriori problemi con loro, basta che questa tristissima storia finisca.

**Reply by gabriele-latino** [reply]
Confermo quanto scritto finora, stanno facendo una figura veramente meschina. Chiedigli se nel bundle della loro visionaria offerta c'e' anche un modem US Robotics da 56k, ed un Pentium 133Mhz, sai mai...
A parte tutto, con 2 installazioni totali, non credo proprio che esista la possibilita' anche remota di aver procurato un danno da sovraccarico ai loro server.
Pensa che io, prima di acquistare un sistema di allarme e quindi spendere alcune migliaia di euro, mi sono assicurato che ci fosse un'integrazione funzionante con Home Assistant. Se questa e' un'azienda che si fregia del fatto di essere italiana e di vendere un prodotto (!) italiano, sono sempre piu' contento di aver dato i miei soldi ad un loro competitor israeliano.

**Reply by maxill1** [reply]
Chiaramente non voglio difendere la scelta di rimuovere l'integrazione, ma se va in polling sul cloud anche solo una volta al minuto per un utente genera un po' meno di 1500 richieste giornaliere verso AWS (l'ip mi sembra sia di amazon) e per loro probabilmente sono costi non preventivati. Stanno a fa un po i purciari, questo si.

**Reply by gabriele-latino** [reply]
Beh ne hanno gia' 1500 al giorno per ogni utente (ipotizzando sempre un polling di 60s), vuol dire che per ogni utente che usa l'integrazione, dovrebbero tenere in conto il doppio delle richieste verso AWS; non calcolano pero' il fatto che magari quell'utente da 3000 richieste al giorno, cosi' facendo se lo sono giocato. Nessun problema, ci saranno piu' utenti per le aziende piu' aperte mentalmente.

**Reply by bigmoby** [reply]
> Chiaramente non voglio difendere la scelta di rimuovere l'integrazione, ma se va in polling sul cloud anche solo una volta al minuto per un utente genera un po' meno di 1500 richieste giornaliere verso AWS (l'ip mi sembra sia di amazon) e per loro probabilmente sono costi non preventivati. Stanno a fa un po i purciari, questo si.

Alibaba Cloud: https://check-host.net/ip-info?host=47.91.74.102 ...ci potrebbe anche stare il tema dei costi non previsti ma...mah! Se facessi installare la mobile app ai miei figli significa che si causerebbe lo stesso "danno"?!

**Reply by maxill1** [reply]
Ah pensavo che anche la versione XR usasse lo stesso ip di quello "normale" (https://check-host.net/ip-info?host=52.28.104.204).

Mi dispiace per chi ha XR, avevo sperato fosse migliore della vecchia centralina, che ha i suoi bei difetti nel rilevare i sensori...

... a questo punto ho fatto bene a rifiutare l'upgrade da iAlarm a iAlarm-XR (me lo avevano proposto qualche tempo fa per un centinaio di euro, riconsegnando la "vecchia")

P.S
per il tuo progetto, se ti servissero info per lo stato dei sensori (messaggio GetByWay), chiedi pure o dai un occhiata alla funzione getZoneStatus di https://github.com/maxill1/node-ialarm/blob/master/src/status-decoder.js

**Reply by bigmoby** [reply]
> 
> P.S per il tuo progetto, se ti servissero info per lo stato dei sensori (messaggio GetByWay), chiedi pure o dai un occhiata alla funzione getZoneStatus di https://github.com/maxill1/node-ialarm/blob/master/src/status-decoder.js

Perfetto, grazie mille! 🙌

**Comment by m1k83** [comment]
Ciao @bigmoby e a tutti gli altri.

Come già detto da altri, l'azienda in questione non poteva comportarsi peggio.
Da sistemista posso solo dire che bloccare il progetto e **inventarsi** dei problemi sui server con due installazioni attive mi sembra veramente ridicolo.

E pensare che li scelsi proprio perchè c'era l'integrazione su HomeAssistant!

Che poi, usano roba **cinese** marchiata con il loro logo, cercate "Meian alarm" sui vari siti di shopping (cinesi e non) e vedrete gli stessi panneli, sensori e sirene, a prezzi ovviamente molto più bassi!

Possedendo una centrale di qualche anno, avevo fatto un mezzo pensiero di aggiornarla con la XR, ma leggendo tutto questo non credo l'upgrade avverrà mai.

Giusto per "informazione", le nuove centrali XR non hanno il protocollo xml cifrato? Neanche l'invio dei messaggi CID (rete CMS)? E neanche da HTTP è possibile controllarle?

Chiedo in quanto al momento uso un'integrazione custom (scritta male in due giorni) che triangola i messaggi xml cifrati, i messaggi CID e polla ogni tot sulla pagina http, tutto rigorosamente in locale senza far uscire la centrale su internet (routing bloccato per quell'ip).

@bigmoby ancora grazie per il tuo lavoro

Saluti

**Reply by bigmoby** [reply]
Ciao @michele-83 ti ringrazio molto per il supporto 👍
Concordo con tutto quanto dici!
Una sola cosa voglio aggiungere, una cosa che reputo di estrema importanza per tutti...il protocollo non è cifrato. Un semplice xor su una stringa non è una cifratura valida e la centrale XR va solo in cloud.
Intelligenti pauca.

**Reply by m1k83** [reply]
> Ciao @michele-83 ti ringrazio molto per il supporto +1 Concordo con tutto quanto dici! Una sola cosa voglio aggiungere, una cosa che reputo di estrema importanza per tutti...il protocollo non è cifrato. Un semplice xor su una stringa non è una cifratura valida e la centrale XR va solo in cloud. Intelligenti pauca.

@bigmoby super gentile!
Effettivamente mi sono espresso male, non è cifrato ma xor-ato con una stringa hardcodata in tutte le centrali e basta. 
Questo è il grado di "sicurezza" made in China. 

Proprio per questo motivo la mia centrale è bloccata in uscita su internet e consiglio a tutti di farlo.

Da quello che mi dici quindi deduco che la XR non accetta nessuna connessione in ingresso, ma è lei che si connette ai suoi server hardcodati nel firmware e fine. 

In questo caso, se un giorno falliscono e chiudono il servizio, quella centrale diventa un utile fermacarte.

Un pò come un'altra marca italiana molto più famosa dove paghi un canone mensile: se chiudi il contratto addio controllo remoto....  

Grazie ancora!

**Reply by bigmoby** [reply]
> Proprio per questo motivo la mia centrale è bloccata in uscita su internet e consiglio a tutti di farlo.

Esattamente!!! 😉 

> Da quello che mi dici quindi deduco che la XR non accetta nessuna connessione in ingresso, ma è lei che si connette ai suoi server hardcodati nel firmware e fine.

Si e purtroppo non avendo neanche più la centrale XR in casa non riesco neanche più a svilupparci sopra per provare a fare un local polling sempre se sia possibile farlo.

> In questo caso, se un giorno falliscono e chiudono il servizio, quella centrale diventa un utile fermacarte.

Di più, un quadretto a muro! 😅

🙌

**Comment by SmartM-ui** [comment]
Caro @bigmoby,
rinnovo i complimenti alla tua persona e al progetto. 

Quelli di Antifurto365, invece di ringraziarti per esser riuscito, a titolo gratuito, ad integrare la loro centrale nel mondo di Home Assistant, ti fanno la guerra.

Anche io avevo comprato la loro centrale perchè mi avevano assicurato fosse integrabile in HA fin da subito e poi, dopo rilascio di aggiornamento con Alexa e IFTTT; aggiornamento che sarebbe dovuto arrivare il mese dopo il mio acquisto. 
...finalmente dopo un anno hanno rilasciato la possibilità di integrarla con Alexa e IFTTT (che nel frattempo è diventato a pagamento oltre le 3 applet) e il suo funzionamente è a dir poco ridicolo per una centrale antifurto, per gestire le automazioni. 
Ritardi fino a 60 minuti!

Sicuramente non consiglio la loro centrale e, appena troverò qualcosa di meglio e integrabile in HA, la sostituirò.

Grazie ancora di tutto BigMoby

**Reply by bigmoby** [reply]
Grazie davvero @SmartM-ui 🙏

**Comment by M4Massi** [comment]
Salve, ho contattato l'assistenza di antifurto365 informandoli che sono in procinto dell'acquisto di un sistema di antifurto e che stavo valutando il loro ialarm-xr perché supportava HA. Inoltre gli ho chiesto chiarimenti riguardo alla rimozione dell'integrazione e gli ho fatto presente che senza in supporto ad HA mi sarei orientato su altro. La loro risposta è stata:

> stiamo sviluppando internamente l'integrazione stabile con home-assistant che sarà pronta a settembre.

Se fosse vero, sarebbe un'ottima notizia. Per ora sono solo promesse, quindi aspettiamo e vediamo se effettivamente la svilupperanno.

**Reply by bigmoby** [reply]
Ciao @M4Massi , al momento non ci sono PR aperte su HA e solitamente, dati i tempi medio-lunghi di approvazione, dubito si possa fare così in fretta, ovviamente posso sbagliarmi. A questo punto sarei molto curioso di vedere il loro codice...attenderò!🍿

**Reply by SmartM-ui** [reply]
> Ciao @M4Massi , al momento non ci sono PR aperte su HA e solitamente, dati i tempi medio-lunghi di approvazione, dubito si possa fare così in fretta, ovviamente posso sbagliarmi. A questo punto sarei molto curioso di vedere il loro codice...attenderò!🍿

Ho il presentimento che sarà uguale al tuo :-)

**Comment by sdavides** [comment]
Ciao a tutti non riesco a connettermi con questo addon, ma mi connetto con quello ufficiale sulla porta 8000, ho descritto qui:
https://github.com/indomus/forum/discussions/239#discussioncomment-3951917

Qualche idea?

Grazie
